src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-tabs/GeneViewTabs.scss

```diff
@@ -7,6 +7,7 @@
   border-radius: 3px;
   margin-right: 3px;
   height: 30px;
+  padding-top: 5px;
```

Comment:
Let's do vertical centering using `display: flex; align-items: center`? It won't care if button size changes in the future.